src/doc/src/reference/publishing.md

```diff
@@ -240,8 +240,23 @@ the “Grant Access” button next to its name:
 
 ![Authentication Access Control](../images/auth-level-acl.png)
 
+#### could not find the github team
```

Comment:
This looks a little strange starting with a lowercase letter.

Reply:
So, "\`could not find the github team\` error" is fine?

Reply:
Maybe "Troubleshooting GitHub team access errors"?